frankenphp_test.go

```diff
@@ -107,19 +107,36 @@ func runTest(t *testing.T, test func(func(http.ResponseWriter, *http.Request), *
 	wg.Wait()
 }
 
+func testRequest(req *http.Request, handler func(http.ResponseWriter, *http.Request)) (string, *http.Response) {
+	w := httptest.NewRecorder()
```

Comment:
Maybe pass `t` as parameter to call [`t.Helper()`](https://pkg.go.dev/testing#T.Helper) ?